react/Popover/Popover.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from 'react'
+import ReactDOM from 'react-dom'
+import PropTypes from 'prop-types'
+export default class Popover extends Component {
+  constructor(props) {
+    super(props)
+    this.element = ''
+    this.state = {
+      style: { display: 'block', visibility: 'hidden', position: 'fixed' }
+    }
+  }
+
+  //When the component is ready, than let's do some Maths
+  componentDidMount() {
+    requestAnimationFrame(this.computeCoordinates.bind(this))
+  }
+  computeCoordinates() {
+    const { refParent, style } = this.props
+    // eslint-disable-next-line
+    const menuElement = ReactDOM.findDOMNode(refParent)
+    const scrollTop = this.getScrollParent(menuElement)
+    const coordinates = menuElement.getBoundingClientRect()
+
+    //Find me & give me my coordinates
+    // eslint-disable-next-line
+    const domItem = ReactDOM.findDOMNode(this.element)
```

Comment:
I think there is no need for the ref `this.element`. `ReactDOM.findDOMNode(this)` would work fine IMHO.